osbs/schemas/container.json

```diff
@@ -106,6 +109,15 @@
       "additionalProperties": false,
       "deprecated": true
     },
+    "buildtime_limit": {
```

Comment:
https://json-schema.org/understanding-json-schema/reference/numeric.html#range we can also do minimum here to avoid negative values

Reply:
added